### DISCUSSION TITLE
Pre-run of ISCE processing Error

### DISCUSSION THREAD

**Discussion by hariiirs** [alexisInSAR/EZ-InSAR#66] [Help]
All steps till 'Select the best reference date' performed well. Now when running 'Pre-run of ISCE Processing', processing stops with the the following message-  
![Screenshot from 2023-10-16 19-28-21](https://github.com/alexisInSAR/EZ-InSAR/assets/42439461/e1e8b8c9-57b0-4649-a06f-ea696e693242)
![Screenshot from 2023-10-16 19-25-31](https://github.com/alexisInSAR/EZ-InSAR/assets/42439461/30e2d144-51a9-43ca-bb7c-bbbda9832c98).

What may be the possible solution for this?

**Comment by alexisInSAR** [comment]
Hi, 

I think you have reproduced an error that we have sometimes encountered. What happens if you restart EZ-InSAR? 

Kind regards, 

Alexis

**Comment by hariiirs** [comment]
Now it is resolved. Actually opencv was not installed in my system. pip3 install opencv-python

Thank you

**Comment by alexisInSAR** [comment]
Perfect. 

Alexis
